### PR TITLE
Feature/rh 57 central UI and navigation

### LINKED ISSUE #57: create central UI to navigate search, map and profile page
As a user, I want to enter a central webpage to navigate between the map, the search and my own profile page to find the functionality I want to use. 

- [x] start page = search view: search bar and button to open my own profile editing page
- [x] map view: show map of HPI
- [x] navigation bar at the bottom: links to search and map, shows icon and "search", "map"
- [x] the buttons are indicating which view is enabled
- [x] Icon that links to own profile show page

### PR DESCRIPTION
**Pull request by TheGrayStone**
Closes #57

### CODE REVIEW

File: app/controllers/welcome_controller.rb [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
-  before_action :authenticate_user!, only: [:protected]
-  # Alternative: before_action :authenticate_user!, :except => [:index]
-
   def index
     # Welcome page, accessible without login
   end
 
-  def protected
+  def login
     # Only accessible by logged in users, see `before_action` call
```
Comment (Olliwehr) [inline app/controllers/welcome_controller.rb:7]: This comment seems to be outdated if the above section about the `before_action` call got deleted, doesn't it?

Overall what do the changes in this file mean in the context of the issue? 🤔

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def route_active_class?(test_path)
+    return 'active' if request.path == test_path
+
+    ''
```
Comment (Olliwehr) [inline app/helpers/application_helper.rb:5]: What are those `''`? 😄

File: app/views/devise/registrations/edit.html.erb [hunk 3]
```diff
@@ -49,6 +49,11 @@
         <%= f.submit "Update User Account", class: 'btn btn-primary' %>
       </div>
     <% end %>
+    <h3 class="fw-light mt-5">Logout Account</h3>
+
+     <%= button_to destroy_user_session_path, class: 'btn btn-outline-primary', method: :delete do %>
+            <i class="fas fa-sign-out-alt"></i> Sign out
```
Comment (Olliwehr) [inline app/views/devise/registrations/edit.html.erb:55]: Is it intended that 'Sign out' is _outside_ the italic-tag here? If so, the tag could be self-closing, right?

File: app/views/welcome/login.html.erb [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<div class="row">
```
Comment (Olliwehr) [inline app/views/welcome/login.html.erb:1]: This file seems to be lacking a bit of correct formatting (e.g. indentations), doesn't it?

**Review by Olliwehr — COMMENTED** [review]
Overall some very nice UI stuff. Got some minor comments , which maybe get resolved very easily ^^

File: app/controllers/welcome_controller.rb (hunk 1 above)
Comment (TheGrayStone) [inline app/controllers/welcome_controller.rb:7]:
```suggestion
```

How 'bout that?

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (TheGrayStone) [reply app/helpers/application_helper.rb:5]: That's a default return, seems to be Ruby style 🤷

File: app/views/devise/registrations/edit.html.erb (hunk 3 above)
Comment (TheGrayStone) [reply app/views/devise/registrations/edit.html.erb:55]: It's a FontAwesome icon 😅

**Review by rfrx — APPROVED** [review]

**Pull request merged**